db/migrate/20151123010203_initial_db.rb

```diff
@@ -8,79 +8,74 @@ def self.up
   end
 
   def self.create_categories
-    unless ActiveRecord::Base.connection.table_exists?('categories')
-      create_table 'categories' do |t|
-        t.string 'name', limit: 255, null: false
-      end
+    return if ActiveRecord::Base.connection.table_exists?('categories')
+    create_table 'categories' do |t|
+      t.string 'name', limit: 255, null: false
     end
   end
 
   def self.create_comments
-    unless ActiveRecord::Base.connection.table_exists?('comments')
-      create_table 'comments' do |t|
-        t.integer  'user_id',  limit: 4,     null: false
-        t.integer  'slide_id', limit: 4,     null: false
-        t.text     'content',  limit: 65535, null: false
-        t.datetime 'created',                null: false
-        t.datetime 'modified'
-      end
-      add_index 'comments', ['slide_id'], name: 'idx_comments_slide_id_key', using: :btree
-      add_index 'comments', ['user_id'], name: 'idx_comments_user_id_key', using: :btree
+    return if ActiveRecord::Base.connection.table_exists?('comments')
+    create_table 'comments' do |t|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->